src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerLivenessProbe.java

```diff
@@ -96,5 +100,69 @@ public String toString() {
         public String getDisplayName() {
             return "Container Liveness Probe";
         }
+
+        public static final String DEFAULT_EXEC_ARGS = "uptime";
+        public static final int    DEFAULT_TIMEOUT_SECONDS = 1;
+        public static final int    DEFAULT_INITIAL_DELAY_SECONDS = 0;
+        public static final int    DEFAULT_PERIOD_SECONDS = 10;
+        public static final int    DEFAULT_FAILURE_THRESHOLD = 3;
+        public static final int    DEFAULT_SUCCESS_THRESHOLD = 1;
+
+        public FormValidation doCheckExecArgs(@QueryParameter String value) {
+            if (StringUtils.isEmpty(value)) {
+                return FormValidation.error("Liveness probe command is mandatory");
+            } else {
+                return FormValidation.ok();
+            }
+        }
+
+        public FormValidation doCheckTimeoutSeconds(@QueryParameter int value) {
```

Comment:
interestingly, if you put a string that isn't a number into the field, the form validation doesn't happen.

Reply:
You can use `<f:number/>` instead of `<f:textbox/>`

Reply:
And use `FormValidation.validatePositiveInteger(value)`

Reply:
interesting, `FormValidation.validatePositiveInteger()` takes a string as the argument. I'd really rather not rework this class to maintain strings and integers for these values.

Reply:
Ah yes because it works from a textbox, so it expects a string as input.

Reply:
The validation method can use string but the rest can use int just fine